boards/common/blxxxpill/include/periph_conf.h

```diff
@@ -310,6 +310,32 @@ static const spi_conf_t spi_config[] = {
 #define SPI_NUMOF           ARRAY_SIZE(spi_config)
 /** @} */
 
+/**
+ * @brief USB device FS configuration
+ */
+static const stm32_usbdev_fs_config_t stm32_usbdev_fs_config[] = {
+    {
+        .base_addr  = (uintptr_t)USB,
+        .rcc_mask   = RCC_APB1ENR_USBEN,
+        .irqn       = USB_LP_IRQn,
+        .apb        = APB1,
+        .dm         = GPIO_PIN(PORT_A, 11),
+        .dp         = GPIO_PIN(PORT_A, 12),
+        .af         = GPIO_AF_OUT_OD,
```

Comment:
```suggestion
        .af         = GPIO_AF_UNDEF,
```
According to the reference manual, these pins are connected automatically to the USB internal transceiver connected to the as soon as the USB is enabled. This change should work with PR #18680.